cmd/clusterctl/client/cluster/proxy_test.go

```diff
@@ -103,6 +103,69 @@ func TestProxyGetConfig(t *testing.T) {
 	})
 }
 
+// These tests are emulating the files passed in via KUBECONFIG env var by
+// injecting the file paths into the ClientConfigLoadingRules.Precedence
+// chain.
+func TestKUBECONFIGEnvVar(t *testing.T) {
```

Comment:
I wrote the tests this way because I preferred to not override the `KUBECONFIG` env var. This method of injecting the precedence path is similar to what is done in kubernetes/client-go.

Reply:
nit. 
```suggestion
func TestKubeconfigEnvVar(t *testing.T) {
```